src/DeleteEntityAction.js

```diff
@@ -0,0 +1,3 @@
+import {createAction} from 'redux-actions';
+
+export default createAction('ENTITY_DELETE', (payload) => payload, (payload, meta) => meta);
```

Comment:
Would be good to have some documentation here describing what the payload looks like. I can see from the implementation and tests that it is keypath that you want to delete but that isn't immediately obvious. Also, is there a way to delete multiple things at once?